stable/elasticsearch-curator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "5.5.4"
 description: A Helm chart for Elasticsearch Curator
 name: elasticsearch-curator
-version: 2.0.2
+version: 2.0.3
```

Comment:
Please bump minor version as it is a new feature.

Reply:
Done, thank you!

Reply:
Have you pushed your changes? I still see the old version.

Reply:
I thought I did, but hadn't - sorry about that. Should be good now 👍